docs/user/metrics/update_downloads.py

```diff
@@ -116,19 +116,9 @@ def condaforge(package: str) -> int:
     Returns:
         Total number of downloads.
     """
-    # TODO: should really be using one of the following instead:
-    # https://github.com/conda-incubator/condastats
-    # https://github.com/anaconda/anaconda-package-data
-    url = f'https://anaconda.org/conda-forge/{package}'
-    pattern = r'<span>(\d+)</span> total downloads'
+    url = f'https://api.anaconda.org/repocore/channels/conda-forge/artifacts/conda/{package}'
```

Comment:
Through a happy accident, conda's website changed and the JavaScript is no longer rendered properly. This breaking change actually led me to discover the actual API being used to populate the webpage. So we no longer need to download the HTML for the entire webpage, we can just query the API directly!